### DISCUSSION TITLE
유니티에서 빌드잇으로 제작한 맵 로드 시 오브젝트의 위치와 회전값이 변경되는 문제

### DISCUSSION THREAD

**Discussion by kimhaech** [naverz/zepeto-studio-kor#334] [5. Q&A]
안녕하세요. 사실 이 문제는 다른 질문으로도 확인했으나 아직 제대로 된 답변을 얻지 못해 작성합니다.
제페토 빌드잇에서 제작한 맵을 유니티로 로드하여 가져오면 몇몇 오브젝트들의 위치와 회전값이 바뀌는 현상이 있습니다.
특별히 한 것 없이 맵만 로드하였습니다. 혹시 해결방안이 있을까요?

**Comment by crash-k** [comment]
안녕하세요.
말씀하신 문제에 대해서 내부에서 파악중에 있는데요.
혹시 가능하시면 그런 현상이 잘 나타나는 맵 코드를 공유 받을 수 있을까요?

감사합니다.

**Reply by kimhaech** [reply]
16586CEC

**Reply by kimhaech** [reply]
여기있습니다 :)

**Comment by kimhaech** [comment]
16586CEC 
제페토 빌드잇 내에서 보이는 이 코드가 맞을까요?
 


 
-----Original Message-----

**Reply by crash-k** [reply]
넵 맞습니다. 공유 감사합니다.
최대한 빠르게 조치 하도록 하겠습니다.

**Comment by crash-k** [comment]
zepeto.buildit.loader@0.0.2 에서 수정되었습니다.

감사합니다.

**Reply by kimhaech** [reply]
감사합니다 :)
